docs/REST/authentication/hmac.md

```diff
@@ -217,6 +217,52 @@ function signRequest(host,
 }
 ```
 
+### React native
```

Comment:
We have a one code sample per programming language in the wiki page. Have you tried using the following JavaScript code sample in your React Native application?

https://github.com/Azure/AppConfiguration/blob/main/docs/REST/authentication/hmac.md#javascript

Reply:
Yes I tried. Since the javascript library for `crypto-js` had to be replaced so I had to update as per the available methods in the new package.
The javascript sample you provide shall work for `nodejs` and `browser`, but for `react-native` it needs another variant.